packages/jaspr_lints/lib/src/assists/css_assist.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/element/type.dart';
+import 'package:analyzer/source/source_range.dart';
+import 'package:analyzer_plugin/protocol/protocol_common.dart';
+import 'package:custom_lint_builder/custom_lint_builder.dart';
+
+import '../utils.dart';
+
+class CssAssistProvider extends DartAssist {
```

Comment:
Every assist class could have some short docs comment with brief description about what exactly does for better understanding..

Reply:
Documented them in the docs